### ISSUE TITLE
A way to blacklist games and remove streams from live window and notifications?

### ISSUE THREAD

**Issue by leandor** [chatty/chatty#72]
Hey there, sorry to bother you with an issue ticket... and I hope I'm not being dumb, but after skimming over the docs on the Live streams and notification settings, I didn't find a way to do what I'd wish to do.

Thing is, I have a large list of twitch streamers I follow (~50) and it always happens when a new "popular" game is released (that I have no interest in, i.e. currently: No Man's Sky) that most/many of my follow list starts playing that game for a period of time, which makes difficult to find streamers that are playing anything that I would want to watch...

Using BetterTTV I can block games in Twitch (and I have blocked several over the past year or so) and the streamers playing those do not appear in the current Live Channels view in the twitch's 'following' section.

What I'd wish is a feature like that, that allows me to block/blacklist games and they get automatically hidden in the live streams view and any status notifications for streams containing such games aren't shown.

Is there a way to do this with the current version? It's something that would be feasible to implement?

Of course, no hurry... just wanted to leave this idea out there, and maybe, if the possibility exists that it could already be implemented and just not documented, just find out how is done.

Thank you very much, and sorry again if it's something that is already present and documented, and I just didn't find it.

**Comment by GhettoGirl** [comment]
I implemented this in my fork. But there is still a problem, the notification sound is still played. So this still needs improvements, but the visuals parts are working.

**Comment by leandor** [comment]
Awesome!  \o/ Looking good :) Looking forward to see what comes up! 

PS: I might try my hand at trying to compile your version myself, so I can test it! 

I'm not proficient with Java, but I've used Eclipse and NetBeans in the past to do minor stuff and I should be able to get the project to compile... Hmmm... maybe tomorrow evening I'd sit down and do that.

Thank you!!

**Comment by GhettoGirl** [comment]
@leandor Just FYI, my fork uses Gradle, you just need to execute `gradle build` and `gradle release` in a terminal or use a Java IDE with Gradle support/plugin. Its that simple. :wink:

**Comment by leandor** [comment]
Yeah, thank you! I was just reading about that... tomorrow is holiday for me, so I'll try to get some testing done in the evening. I'll report back, for sure.
